docs/install/Linux_Install_Guide.rst

```diff
@@ -96,14 +96,14 @@ For Centos, use:
     yum info rocm-libs
 
 The ROCm version has major, minor, and patch fields, possibly followed by a build specific identifier. For example, ROCm version could be 4.0.0.40000-23; this corresponds to major = 4, minor = 0, patch = 0, build identifier 40000-23.
-There are GitHub branches at the rocBLAS site with names rocm-major.minor.x where major and minor are the same as in the ROCm version. For ROCm version 4.0.0.40000-23, you must use the following to download rocBLAS:
+There are GitHub branches at the rocBLAS site with names rocm-major.minor.x where major and minor are the same as in the ROCm version. To download rocBLAS, you can use the following command:
 
 ::
 
    git clone -b release/rocm-rel-x.y https://github.com/ROCm/rocBLAS.git
    cd rocBLAS
 
-Replace x.y in the above command with the version of ROCm installed on your machine. For example, if you have ROCm 5.0 installed, then replace release/rocm-rel-x.y with release/rocm-rel-5.0.
+Replace x.y in the above command with the version of ROCm installed on your machine. For example, if you have ROCm 6.0 installed, then replace release/rocm-rel-x.y with release/rocm-rel-6.0.
```

Comment:
While we are making changes, why not change the release to 6.2?

Reply:
Sure, done.